docs/aws/setup-mainnet-validator-from-scratch.md

```diff
@@ -218,114 +205,11 @@ Get the trusted hash from the network:
-
-Execute the following command to download and stage the upgrade:
-```
-cd ~; curl -sSf genesis.casperlabs.io/casper/1_4_1/stage_upgrade.sh | sudo bash -
-```
-
-#### Upgrade to casper-node v1.4.3
-For this upgrade, to `casper-node v1.4.3`, the activation point is `Era 3111`. You have to make sure you have properly staged the upgrade well ahead of the activation point, so that your node will be upgraded on time.
-
-Execute the following command to download and stage the upgrade:
-```
-cd ~; curl -sSf genesis.casperlabs.io/casper/1_4_3/stage_upgrade.sh | sudo bash -
-```
-#### Upgrade to casper-node v1.4.4
-For this upgrade, to `casper-node v1.4.4`, the activation point is `Era 3435`. You have to make sure you have properly staged the upgrade well ahead of the activation point, so that your node will be upgraded on time.
-
-Execute the following command to download and stage the upgrade:
-```
-cd ~; curl -sSf genesis.casperlabs.io/casper/1_4_4/stage_upgrade.sh | sudo bash -
-```
-
-#### Upgrade to casper-node v1.4.5
-For this upgrade, to `casper-node v1.4.5`, the activation point is `Era 4417`. You have to make sure you have properly staged the upgrade well ahead of the activation point, so that your node will be upgraded on time.
-
-Execute the following command to download and stage the upgrade:
-```
-cd ~; curl -sSf genesis.casperlabs.io/casper/1_4_5/stage_upgrade.sh | sudo bash -
-```
-
-#### Upgrade to casper-node v1.4.6
-For this upgrade, to `casper-node v1.4.6`, the activation point is `Era 4968`. You have to make sure you have properly staged the upgrade well ahead of the activation point, so that your node will be upgraded on time.
-
-Execute the following command to download and stage the upgrade:
-```
-cd ~; curl -sSf genesis.casperlabs.io/casper/1_4_6/stage_upgrade.sh | sudo bash -
-```
-
-#### Upgrade to casper-node v1.4.8
-For this upgrade, to `casper-node v1.4.8`, the activation point is `Era 6130`. You have to make sure you have properly staged the upgrade well ahead of the activation point, so that your node will be upgraded on time.
-
-Execute the following command to download and stage the upgrade:
-```
-cd ~; curl -sSf genesis.casperlabs.io/casper/1_4_8/stage_upgrade.sh | sudo bash -
-```
-
-#### Upgrade to casper-node v1.4.9
-For this upgrade, to `casper-node v1.4.9`, the activation point is `Era 7210`. You have to make sure you have properly staged the upgrade well ahead of the activation point, so that your node will be upgraded on time.
-
-Execute the following command to download and stage the upgrade:
-```
-cd ~; curl -sSf genesis.casperlabs.io/casper/1_4_9/stage_upgrade.sh | sudo bash -
-```
+### Staging the upgrades
+"Staging an upgrade" is a process in which you tell your node to download the upgrade files and prepare them, so that they can automatically be applied at the pre-defined activation point. The ```node_util``` script used earlier will have staged all available upgrades for your node.
```

Comment:
```suggestion
"Staging an upgrade" is a process in which you tell your node to download the upgrade files and prepare them, so that they can automatically be applied at the pre-defined activation point. The same ```node_util``` script used earlier will have stage available upgrades for your node since when you started it.  There is no harm in running it multiple times as it will not re-stage any already loaded.


```